pysite/views/api/bot/starboard.py

```diff
@@ -13,11 +13,11 @@
 })
 
 POST_SCHEMA = Schema({
-    "bot_message_id": str,
-    "message_id": str,
-    "guild_id": str,
-    "channel_id": str,
-    "author_id": str,
+    "bot_message_id": int,
+    "message_id": int,
+    "guild_id": int,
+    "channel_id": int,
+    "author_id": int,
```

Comment:
are you sure about this? I think we used strings here because rethink had some issues storing snowflakes as ints. I could be wrong though. But this should definitely be checked - maybe @heavysaturn or @jos-b know?

Reply:
I believe i had it as strings to begin with, but found it easier to work with ints.  But if they don't work with rethink i can change them back

Reply:
Yeah we definitely had issues with RethinkDB not being able to store snowflakes, these should probably be strings.